util/cli/common_options.go

```diff
@@ -0,0 +1,20 @@
+package cli
+
+import (
+	"github.com/src-d/lookout/util/grpchelper"
+)
+
+type initializer interface {
+	init(*App)
+}
+
+// CommonOptions contains common flags for all commands
+type CommonOptions struct {
+	LogOptions
+	GrpcMaxMsgSize int `long:"grpc-max-message-size" default:"100" env:"LOOKOUT_GRPC_MAX_MSG_SIZE" description:"max. message size to send/receive to/from clients (in MB)"`
```

Comment:
nitpick comment: a shorter name would be nicer in my opinion, e.g. `grpc-max-size` and `LOOKOUT_GRPC_MAX_SIZE`

Reply:
I took the name from bblfshd. Though I prefer a shorter one too. 👍

Reply:
shorter version might be nice, but in this case consistency between project seem more valuable.

Also, there are number of parameters that you can set in gRPC regarding MAX size, and in this case it's only max _message_ size, so it's good